### PR TITLE
Add bidirectional-override

### PR DESCRIPTION
**Pull request by jamesnw**

### CODE REVIEW

File: features/bidirectional-override.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Bidirectional override
```
Comment (autonome) [inline features/bidirectional-override.yml:1]: The title seems ambiguous, maybe should include some combo of `Unicode` or `layout` or `direction` or something like that, which would disambiguate from people thinking it's the `bdo` tag (separate but related feature).

File: features/bidirectional-override.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description: The `unicode-bidi` and `direction` CSS properties override the Unicode layout algorithm in non-HTML documents. Authors should use the `dir` global HTML attribute instead.
```
Comment (autonome) [inline features/bidirectional-override.yml:2]: "in non-HTML documents" doesn't seem right? Do they only apply in that case?

**Review by autonome — COMMENTED** [review]
Some questions

File: features/bidirectional-override.yml (hunk 1 above)
Comment (jamesnw) [reply features/bidirectional-override.yml]: Good point, I renamed "Layout direction override"

File: features/bidirectional-override.yml (hunk 2 above)
Comment (jamesnw) [reply features/bidirectional-override.yml]: They are applied in HTML documents, but use is discouraged. Stylesheets could be disabled, so text direction should be set in the HTML and not in CSS. I updated the description to clarify.

File: features/layout-direction-override.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: Layout direction override
+description: The `unicode-bidi` and `direction` CSS properties override the Unicode layout algorithm. For HTML documents, authors should use the `dir` global HTML attribute and `<bdo>` HTML element instead.
```
Comment (ddbeck) [inline features/layout-direction-override.yml:2]: As written, this sounds like an anti-feature. Is it possible you could link to some history about this feature? I'm wondering who it's _for_, if it's not deprecated.

Comment (jamesnw) [reply features/layout-direction-override.yml]: Good point- I'm not seeing historical posts about this, but both [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/unicode-bidi) and [WebReference](https://webreference.com/css/properties/unicode-bidi/) say it is explicitly for DTD designers, so I updated the description.

File: features/layout-direction-override.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: Layout direction override
+description: The `unicode-bidi` and `direction` CSS properties override the Unicode layout algorithm, and is intended for Document Type Definition (DTD) designers. For HTML documents, authors should use the `dir` global HTML attribute and `<bdo>` HTML element instead.
```
Comment (autonome) [inline features/layout-direction-override.yml:2]:
```suggestion
description: The `unicode-bidi` and `direction` CSS properties override the Unicode layout algorithm, and are intended for Document Type Definition (DTD) designers. For HTML documents, authors should use the `dir` global HTML attribute and `<bdo>` HTML element instead.
```

**Review by autonome — APPROVED** [review]

**Review by ddbeck — APPROVED** [review]
Optional description suggestions for you. In any case, ready to go. Thank you!

**Pull request merged**